components/collection/unlockable/CountdownTimer.vue

```diff
@@ -1,14 +1,15 @@
 <template>
-  <div ref="wrapper" class="px-4 py-4 keen-slider">
+  <div ref="wrapper" class="px-4 py-4 keen-slider background-inverse">
     <div
-      v-for="(section, index) in [1, 2, 3, 4]"
+      v-for="(section, index) in Array(slidesCountOnTimeCountdown).fill(0)"
```

Comment:
I would make it computed value `Array(slidesCountOnTimeCountdown).fill(0)` 
`section` defined but not used